client/modules/IDE/components/Searchbar.jsx

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import InlineSVG from 'react-inlinesvg';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { throttle } from 'lodash';
+import * as SortingActions from '../actions/sorting';
+
+const searchIcon = require('../../../images/magnifyingglass.svg');
+
+class Searchbar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchValue: this.props.searchTerm
+    };
+    this.handleSearchEnter = this.handleSearchEnter.bind(this);
```

Comment:
i don't think you need to call bind for `this.handleSearchEnter` or `this.handleSearchChange`